src/invoke.rs

```diff
@@ -48,8 +47,8 @@ pub struct Cmd {
 pub enum Error {
     #[error("parsing argument {arg}: {error}")]
     CannotParseArg { arg: String, error: StrValError },
-    #[error("parsing XDR argument {arg}: {error}")]
-    CannotParseXDRArg { arg: String, error: XdrError },
+    #[error("parsing XDR key {arg}: {error}")]
+    CannotParseXdrArg { arg: String, error: XdrError },
```

Comment:
```suggestion
    #[error("parsing XDR arg {arg}: {error}")]
    CannotParseXdrArg { arg: String, error: XdrError },
```
Looks like this was maybe changed by mistake?